### PR TITLE
Add > SP Cloud > Integrations Doc > Looker

### PR DESCRIPTION
**Pull request by RahulSrivastav14**

### CODE REVIEW

File: docs/cloud/integrations/looker.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+  ```
+
+## Getting started
+
+Looker is accessible through the GCP console. Here we will create a Steampipe Cloud connection from Looker.
+
+To create a new connection, click on `Database` then `Connections` from the admin page. Then click `Add Connection` and select `PostgreSQL 9.5+` from the `Dialect` dropdown list, enter the Steampipe Cloud connection details. Set the SSL Mode to on form advanced settings and click `Test` to `Verify` then click `Save`.
```
Comment (judell) [inline docs/cloud/integrations/looker.md:39]:
```suggestion
To create a new connection, click on `Database` then `Connections` from the admin page. Then click `Add Connection`,  select `PostgreSQL 9.5+` from the `Dialect` dropdown list, and enter the Steampipe Cloud connection details. In `Additional Settings`, set the SSL Mode to `On`.  Click `Test` to `Verify`, then click `Save`.
```

Comment (judell) [reply docs/cloud/integrations/looker.md:39]: I assume SSL Mode is under Additional Settings? Maybe expand to show that?

File: docs/cloud/integrations/looker.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Here we use the GCP schema and query the `gcp_compute_region` table. Looker displays the table's schema and previews its data. You can export the data to JSON, XML, TXT, HTML, CSV or Markdown format.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/looker-region-data-preview.png" />
+</div>
+
+## Create a report dashboard to analyze resources
+
+We'll focus here on creating a dashboard to monitor and analyze GCP Storage resources. To begin, create a new project and select `Create a New Model`, then select the steampipe cloud database and schema from the `Allowed Connections` and save. The new model can be accessed through the `Explore` tab.
```
Comment (judell) [inline docs/cloud/integrations/looker.md:59]:
```suggestion
We'll focus here on creating a dashboard to monitor and analyze GCP Storage resources. To begin, create a new project and select `Create a New Model`, then select the Steampipe Cloud database and schema from the `Allowed Connections` and save. The new model can be accessed through the `Explore` tab.
```

**Review by judell — CHANGES_REQUESTED** [review]
just minor tweaks

File: docs/cloud/integrations/looker.md (hunk 1 above)
Comment (RahulSrivastav14) [reply docs/cloud/integrations/looker.md:39]: I've updated the Image as well that shows the SSL slider.

**Review by judell — APPROVED** [review]

**Pull request merged**